source/includes/connect/atlas-connection.java

```diff
@@ -0,0 +1,44 @@
+import com.mongodb.*;
```

Comment:
nit: Generally * imports are frowned upon as they are greedy and can hide subtle bugs.